src/isPalindrome.js

```diff
@@ -19,7 +19,10 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  let check = /[\W]/g;
+  let newStr = str.toLowerCase().replace(check, '');
```

Comment:
check and newStr are way too generic names. It's not clear what is stored in those variables. Variable name should indicate what value it stores.